versions/3.1.1.md

```diff
@@ -211,13 +213,15 @@ Types that are not accompanied by a `format` property follow the type definition
 
 The formats defined by the OAS are:
 
-| [`type`](#data-types) | [`format`](#data-type-format) | Comments                     |
-| -------------------- | --------------------------- | ---------------------------- |
-| `integer`            | `int32`                     | signed 32 bits               |
-| `integer`            | `int64`                     | signed 64 bits (a.k.a long)  |
-| `number`             | `float`                     |                              |
-| `number`             | `double`                    |                              |
-| `string`             | `password`                  | A hint to obscure the value. |
+| [Data Model Type](#data-types) | [`format`](#data-type-format) | Comments                     |
+| ------------------------------ | ----------------------------- | ---------------------------- |
+| number                         | `int32`                       | signed 32 bits               |
+| number                         | `int64`                       | signed 64 bits (a.k.a long)  |
+| number                         | `float`                       |                              |
+| number                         | `double`                      |                              |
+| string                         | `password`                    | A hint to obscure the value. |
+
+As noted above, both `type: number` and `type: integer` are considered to be numbers in the data model.
```

Comment:
I think this table should be "flipped around" to better align with the format registry. Also changed to "primitive type" to align with the change above.
```suggestion
| [`format`](#data-type-format) | [primitive type](#data-types) | Comments                     |
| ------------------------------ | ----------------------------- | ---------------------------- |
| `int32`                      | number                         | signed 32 bits               |
| `int64`                      | number                          | signed 64 bits (a.k.a long)  |
| `float`                       | number                         |                              |
| `double`                      | number                        |                              |
| `password`                    | string                         | A hint to obscure the value. |

As noted above, both `type: number` and `type: integer` are considered to be numbers in the data model.
```

Reply:
@mikekistler #4015 uses "Instance Type": why not use the same terminology here?

Reply:
Only because I slept in between making comments there and here ;-). I agree that we should make the terminology consistent and I think "Instance Type", with a brief explanation of the term at it first appearance, is now my preference. But open to other opinions.

Reply:
I thought I had updated this yesterday to say "JSON Type" but I guess maybe I just made a comment about it on one of the other 9000 issues or PRs open on this topic ;-)  I do not think "instance" is intuitive for most people.